app/views/unattended/provisioning_templates/registration/global_registration.erb

```diff
@@ -25,8 +25,13 @@ if [ -f /etc/os-release ] ; then
   . /etc/os-release
 fi
 
+SSL_CA_CERT=$(mktemp --suffix .crt)
+tee $SSL_CA_CERT << CA
+<%= foreman_ssl_certificate %>
```

Comment:
Can we use the `save_to_file` macro?

Reply:
We could do something like this:
```ruby
<%
ca_cert = "/tmp/foreman_#{rand_hex(5)}.crt"
%>

<%= save_to_file(ca_cert, foreman_ssl_certificate) %>

curl --cacert <%= ca_cert %> --request POST
```
But that's the same as now, but just with more ruby.

Reply:
I personally prefer the shell way, that's easier to read and comprehend. I was just wondering, we might have wanted to use our own macros here (dogfooding).